common/test/acceptance/tests/video/test_video_license.py

```diff
@@ -84,8 +84,11 @@ def test_arr_license(self):
         self.lms_courseware.visit()
         video = self.lms_courseware.q(css=".vert .xblock .video")
         self.assertTrue(video.is_present())
-        video_license = self.lms_courseware.q(css=".vert .xblock.xmodule_VideoModule .xblock-license")
-        self.assertTrue(video_license.is_present())
+        video_license_css = ".vert .xblock.xmodule_VideoModule .xblock-license"
```

Comment:
Was the failure that the element was not present, or that the text was wrong?

If it was that the text was wrong (and it is expected that the text changed from some other value to this new value), the best practice is to wait for the text to switch to what is expected.